meta-balena-common/recipes-support/resin-init/resin-init-flasher/resin-init-flasher

```diff
@@ -127,7 +127,7 @@ fi
 info "resin-init-flasher configuration found."
 
 # Find path to image
-_balena_image=$(find / -xdev -type f -name "${BALENA_IMAGE}")
+_balena_image=$(find / /tmp -xdev -type f -name "${BALENA_IMAGE}" | tail -1)
```

Comment:
Why tail -1? That looks fragile, are we expecting to find more than one image? And why the need to add `/tmp`? is the xdev preventing zram mounts to be searchable as part of `/`?

Reply:
@alexgg Yes, exactly. The `-xdev` parameter prohibits descending into directories on other
filesystems, and `/tmp` is a separate filesystem on zram. Adding it as another search target allows matches to be returned from that directory, but because it's a subdirectory of the root, two results are then returned, i.e.
```
/tmp/balena-image-generic-amd64.balenaos-img
/tmp/balena-image-generic-amd64.balenaos-img
```
Without parsing, this ends up failing the existence check below.

Reply:
@jakogut I'd rather do it in two passes then:

_balena_image=$(find / -xdev -type f -name "${BALENA_IMAGE}")
if [  ! -f "${_balena_image}" ]; then
    _balena_image=$(find /tmp -xdev -type f -name "${BALENA_IMAGE}")
    if [ ! -f "${_balena_image}" ]; then
        fail "Raw image ${BALENA_IMAGE}  not found in rootfs"
    fi
fi

Reply:
Looks good to me. Tested with both flashing and migrating, works great.